packages/react/src/components/Provider/ProviderConsumer.tsx

```diff
@@ -2,15 +2,15 @@ import * as PropTypes from 'prop-types'
 import * as React from 'react'
 import { FelaTheme } from 'react-fela'
 
-import { ThemePrepared } from '../../themes/types'
+import { ProviderContextPrepared } from '../../themes/types'
 
 export interface ProviderConsumerProps {
   /**
    * Uses the function children pattern to access theme.
    * @param {object} theme
    * @param {object} theme.siteVariables - The siteVariables passed from the nearest Provider.
    */
-  render: (theme: ThemePrepared) => React.ReactNode
+  render: (context: ProviderContextPrepared) => React.ReactNode
```

Comment:
would suggest to leave the `render` prop's function signature the same: essentially, we would like client to use only the `theme` attributes here. This will allow us to avoid lots of related changes that had resulted in decomposition of even more deeply nested objects:

```jsx
 <ProviderConsumer
      render={({
        theme: {
          siteVariables: { colors, emphasisColors, naturalColors, transparentColors },
        },
      }) => (
```

Reply:
Do you suggest that we should have the rtl, renderer and disableAnimations props here as part of the team object?

Reply:
no, the structure of `context` object is good now 👍 It is just about changing the argument provided for this particular `render` method of `ProviderConsumer`

Reply:
It's resolved. Please take a look